.github/workflows/ci.yml

```diff
@@ -39,23 +39,31 @@ jobs:
       fail-fast: ${{ startsWith(github.ref, 'refs/tags/') }}
       matrix:
         include:
+          - os: windows-latest
+            platform: windows
+            arch: x86_64
+            toolchain: nightly-x86_64-pc-windows-gnu
           - os: ubuntu-latest
             platform: linux
             arch: x86_64
+            toolchain: nightly
           - os: ubuntu-latest
             platform: linux
             arch: aarch64
+            toolchain: nightly
           - os: macos-latest
             platform: darwin
             arch: x86_64
```

Comment:
It appears that toolchain is not specified.

Reply:
The default is nightly, but it is not needed here because the specific toolchain is set in the makefile

Reply:
I see, you are indeed running `rustup` at the beginning of `Run Tests`.

Reply:
For other platforms, the default stable/nightly is OK. This is mainly for using the gnu toolchain on the windows platform to be able to build c related code